modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/UserResourceManagerTest.java

```diff
@@ -79,14 +327,17 @@ public void tearDown() {
         String id = user.getId();
 
         return new Object[][]{
-                {id, "userName", null, 200, user}
+                {id, null, null, user},
+                {id, null, "emails", user},
+                {id, "userName,meta", null, user},
+                {id, "userName", "emails", user}
```

Comment:
can also consider datasets with empty strings for `attributes` and `excludeAttributes`

Reply:
add a dataset where `excludeAttributes` is a comma separated string

Reply:
what's the difference between the method here and the method in https://github.com/wso2/charon/pull/335/files#diff-32914d3be281f9ccb36fe8cc05d50a694f7364dbf5b6ba7623f7d29103998f38R373

Reply:
Added in commit 6f5ca461fa07ac4b49182320325156a58b80b910